docs/cas-server-documentation/installation/Rest-Authentication.md

```diff
@@ -45,6 +45,20 @@ Expected responses from the REST endpoint are mapped to CAS as such:
 | `428`          | Produces a `AccountPasswordMustChangeException`
 | Other          | Produces a `FailedLoginException`
 
+## Warnings
+
+The remote REST endpoint can send warnings back to the CAS server using custom headers:
```

Comment:
You are not necessarily sending warnings. You're collecting messages, some of which may act as a warning. I think a better header name for this section might be more fitting.

Reply:
The messages are rendered under the headline **_Authentication Succeeded with Warnings_** and will be styled by the `alert alert-warning`. Therefore I thought warnings is the correct word here.

Reply:
Makes sense. Thank you.